content-repo/test_data/circleci-forked-build.json

```diff
@@ -397,7 +397,7 @@
install pipenv\n#             pipenv install\n#       - run:\n#           name: NPM Install\n#           command: |\n#             nvm use\n#             node --version\n#             npm --version\n#             npm ci\n#       - run:\n#           name: NPM Build content-repo docs\n#           command: |\n#             nvm use\n#             npm run reference-docs\n#       - run:\n#           name: NPM Build\n#           no_output_timeout: 40m\n#           environment:\n#             ALGOLIA_INDEX: demisto\n#           command: |\n#             nvm use            \n#             npm run heapstats\n#             npm run build-docusaurus\n#             echo \\\"Build is done. Create tar of the build dir\\\"\n#             tar czf build-site.tar.gz build\n#       - store_artifacts:\n#           path: build-site.tar.gz\n#       - save_cache:\n#           key: npm-cache-{{ checksum \\\"package-lock.json\\\" }}\n#           paths:\n#             - node_modules\n#       - persist_to_workspace:\n#           # Must be an absolute path, or relative path from working_directory. This is a directory on the container which is \n#           # taken to be the root directory of the workspace.\n#           root: /home/circleci/project\n#           # Must be relative path from root\n#           paths:\n#             - build \n# \n#   deploy:\n#     docker:\n#       - image: circleci/python:3.7.9-buster-node\n#     resource_class: small\n#     steps:\n#       - checkout\n#       - attach_workspace:\n#           at: /home/circleci/project\n#       - run:\n#           name: Setup Python\n#           command: |            \n#             pipenv install\n#       - run:\n#           name: Deploy to Firebase\n#        no_output_timeout: 30m\n#        command: |\n#          echo $FIREBASE_JSON_KEY > \"$HOME\"/gcloud.json\n#          npm install firebase-tools\n#          export PATH=\"$PATH:`pwd`/node_modules/.bin\"\n#          export GOOGLE_APPLICATION_CREDENTIALS=\"$HOME\"/gcloud.json\n#          if [ \"$CIRCLE_BRANCH\" = \"master\" ]; then\n#            echo \"Deploying to prod as CIRCLE_BRANCH is equal to master!\"\n#            firebase deploy --only hosting > deploy-info-firebase.txt\n#          else\n#            # Get the PR number from the CIRCLE_PULL_REQUEST env variable which is in the form of:\n#          # https://github.com/demisto/content-docs/pull/335\n#            firebase hosting:channel:deploy \"pull-request-${CIRCLE_PULL_REQUEST##*/}\" --only xsoar.pan.dev > deploy-info-firebase.txt\n#          cat deploy-info-firebase.txt\n#          fi\n#    - run:\n#        name: Post Comment Firebase\n#        command: |\n#          pipenv run ./content-repo/post_github_comment.py deploy-info-firebase.txt\n# \n# workflows:\n#   build_and_deploy:\n#     jobs:\n#       - build\n#       - deploy:\n#           requires:\n#             - build\n#           filters:\n#             branches:\n#               # only from non-fork\n#               only: /^(?!pull\\\\/).*$/"
```

Comment:
this is hard to read, what changed here? same for the rest

Reply:
This is the circle ci config yml, since it was updated I updated it here